src/renderer/components/common/QrCode/common/constants.ts

```diff
@@ -36,8 +36,14 @@ const SEED_INFO: Codec<SeedInfo> = object(
   ['derivedKeys', array(ADDRESS_INFO)],
 );
 
+const FEATURE = taggedUnion('VaultFeatures', [['BulkOperations'], ['DynamicDerivations']]);
+
 // Export address format for decoding; Rust enum is a tagged union
 export const EXPORT_ADDRESS = taggedUnion('ExportAddrs', [['V1', ['payload', array(SEED_INFO)]]]);
+export const EXPORT_ADDRESS_2 = taggedUnion('ExportAddrs', [
+  ['V1', ['payload', array(SEED_INFO)]],
+  ['V2', ['payload', object(['addr', SEED_INFO], ['features', array(FEATURE)])]],
+]);
```

Comment:
Why do we need a separate type for that? `EXPORT_ADDRESS_2` should decode `EXPORT_ADDRESS` payloads as well because its `V1` variant is exactly the same